app/controlplane/api/buf.gen.yaml

```diff
@@ -16,7 +16,7 @@ plugins:
   - name: validate
     out: .
     opt: paths=source_relative,lang=go
-  - remote: buf.build/stephenh/plugins/ts-proto
+  - plugin: buf.build/community/stephenh-ts-proto:v1.151.1
```

Comment:
I needed to change this because of https://buf.build/docs/migration-guides/migrate-remote-generation-alpha/